expected/15.0/sqlite_fdw.out

```diff
@@ -1834,7 +1860,2433 @@ DROP FOREIGN TABLE grem1_1;
+ERROR:  character with byte sequence 0xd0 0x93 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+INSERT INTO "Unicode data" (i, t) VALUES ('bel+', 'У рудога вераб’я ў сховішчы пад фатэлем ляжаць нейкія гаючыя зёлкі._');
+ERROR:  character with byte sequence 0xd0 0xa3 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+SELECT * FROM "Unicode data" WHERE i = 'bel+';
+ERROR:  invalid byte sequence for encoding "UTF8": 0xa7
+DELETE FROM "Unicode data" WHERE t = 'У рудога вераб’я ў сховішчы пад фатэлем ляжаць нейкія гаючыя зёлкі._';
+ERROR:  character with byte sequence 0xd0 0xa3 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+INSERT INTO "Unicode data" (i, t) VALUES ('bul+', 'Ах, чудна българска земьо, полюшвай цъфтящи жита._');
+ERROR:  character with byte sequence 0xd0 0x90 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+SELECT * FROM "Unicode data" WHERE i = 'bul+';
+ERROR:  invalid byte sequence for encoding "UTF8": 0xa7
+DELETE FROM "Unicode data" WHERE t = 'Ах, чудна българска земьо, полюшвай цъфтящи жита._';
+ERROR:  character with byte sequence 0xd0 0x90 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+INSERT INTO "Unicode data" (i, t) VALUES ('rus+', 'Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства._');
+ERROR:  character with byte sequence 0xd0 0xa8 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+SELECT * FROM "Unicode data" WHERE i = 'rus+';
+ERROR:  invalid byte sequence for encoding "UTF8": 0xa7
+DELETE FROM "Unicode data" WHERE t = 'Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства._';
+ERROR:  character with byte sequence 0xd0 0xa8 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+INSERT INTO "Unicode data" (i, t) VALUES ('ukr+', 'Гей, хлопці, не вспію — на ґанку ваша файна їжа знищується бурундучком._');
+ERROR:  character with byte sequence 0xd0 0x93 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+SELECT * FROM "Unicode data" WHERE i = 'ukr+';
+ERROR:  invalid byte sequence for encoding "UTF8": 0xc3 0xe5
+DELETE FROM "Unicode data" WHERE t = 'Гей, хлопці, не вспію — на ґанку ваша файна їжа знищується бурундучком._';
+ERROR:  character with byte sequence 0xd0 0x93 in encoding "UTF8" has no equivalent in encoding "WIN1256"
+-- 1256
+SELECT * FROM "Unicode data" WHERE i = 'ara';
+  i  |                  t                  
+-----+-------------------------------------
+ ara | أبجد هوَّز حُطّي كلَمُن سَعْفَص قُرِشَت ثَخَدٌ ضَظَغ
+(1 row)
+
+SELECT * FROM "Unicode data" WHERE t LIKE '%ضَظَغ%';
+  i  |                  t                  
+-----+-------------------------------------
+ ara | أبجد هوَّز حُطّي كلَمُن سَعْفَص قُرِشَت ثَخَدٌ ضَظَغ
+(1 row)
+
+INSERT INTO "Unicode data" (i, t) VALUES ('ara+', 'أبجد هوَّز حُطّي كلَمُن سَعْفَص قُرِشَت ثَخَدٌ ضَظَغ_');
+SELECT * FROM "Unicode data" WHERE i = 'ara+';
+ERROR:  invalid byte sequence for encoding "UTF8": 0xc3 0xc8
```

Comment:
In previous TC, example:
SELECT * FROM "Unicode data" WHERE i = 'ara';
We can select data for ARA, but after insert data for this encoding, we can not select data. 

How do you think? you handle this case for insert or select.

Similar situation for other encoding, please check all.

Reply:
I have found and fixed code without text conversion. See in https://github.com/pgspider/sqlite_fdw/pull/76/commits/f830585213459aaafc2af0fed690fe6c604813c9 Thanks for good testcase idea, @bichht0608 !

Reply:
@mkgrgis  it seem be good fix. How to ensure for DELETE case? I think similar as INSERT, you should add some SELECT cases after DELETE to ensure about DELETE implementation (not need to add all SELECT after DELETE). How do you think?

Reply:
Thanks for good idea, @bichht0608 ! `SELECT count(*) FROM "Unicode data" WHERE i = '...+';` really tests `sqlite_fdw` context, not doubles ICU or `pg_do_encoding_conversion` tests. I'll add the new testcases.

Reply:
Added in https://github.com/pgspider/sqlite_fdw/pull/76/commits/22b6c5c417475ab0e17a43251e87f8546370d7c0 , @bichht0608 !